src/Sensors.hpp

```diff
@@ -144,6 +148,7 @@ class Sensors {
     float getGas();
 
     void setTempOffset(float offset);
+    void setCO2AltitudeCompensation(float alt_comp);
```

Comment:
please use the same name of parameter used in the implementation, like this:

`void setCO2AltitudeCompensation(float altitude);`

Reply:
done